src/specs/concepts/permissions_concepts.cpt

```diff
@@ -1,16 +1,70 @@
+* Restart network <id>
+* Deploy "storec" smart contract with initial value "5" from a default account in "Node1" until block height reaches <qip714block>
+
+# Validate network admin role
+* Validate that org "NWADMIN" is approved, has "Node1" linked and has role "NWADMIN"
+* Validate that org "NWADMIN" is approved, has "Node2" linked and has role "NWADMIN"
+* Validate that org "NWADMIN" is approved, has "Node3" linked and has role "NWADMIN"
+* Check "Node1"'s default account is from org "NWADMIN" and has role "NWADMIN" and is org admin and is active
+
+# Propose new organization to the network <version> <id>
+* Validate that org "NWADMIN" is approved, has "Node1" linked and has role "NWADMIN"
+* Check "Node1"'s default account is from org "NWADMIN" and has role "NWADMIN" and is org admin and is active
+* From "Node1" propose and approve new org "ORG1" with "Node4"'s enode id and "Default" account
+* Start stand alone "Node4"
+* Write "permissionsConfig" to the data directory of "Node4"
+* Restart network <id>
+* Validate that org "ORG1" is approved, has "Node4" linked and has role "ORGADMIN"
+
+# Suspend an org <version> <id>
+* Validate that org "NWADMIN" is approved, has "Node1" linked and has role "NWADMIN"
+* Check "Node1"'s default account is from org "NWADMIN" and has role "NWADMIN" and is org admin and is active
+* From "Node1" propose and approve new org "ORG1" with "Node4"'s enode id and "Default" account
+* Start stand alone "Node4"
+* Write "permissionsConfig" to the data directory of "Node4"
+* Restart network <id>
+* Validate that org "ORG1" is approved, has "Node4" linked and has role "ORGADMIN"
+* From "Node1" suspend org "ORG1", confirm that org status is "PendingSuspension"
+* From "Node1" approve org "ORG1"'s suspension, confirm that org status is "Suspended"
+* Deploy "storec" smart contract with initial value "5" from a default account in "Node4" fails with error "read only account. cannot transact"
+* From "Node1" revoke suspension of org "ORG1", confirm that org status is "RevokeSuspension"
+* From "Node1" approve org "ORG1"'s suspension revoke, confirm that org status is "Approved"
+* Deploy "storec" smart contract with initial value "5" from a default account in "Node4", named this contract as "c1"
+* "c1"'s "getc" function execution in "Node4" returns "5"
+
+# Deactivate an org <version> <id>
+* Validate that org "NWADMIN" is approved, has "Node1" linked and has role "NWADMIN"
+* Check "Node1"'s default account is from org "NWADMIN" and has role "NWADMIN" and is org admin and is active
+* From "Node1" propose and approve new org "ORG1" with "Node4"'s enode id and "Default" account
+* Start stand alone "Node4"
+* Write "permissionsConfig" to the data directory of "Node4"
+* Restart network <id>
+* Validate that org "ORG1" is approved, has "Node4" linked and has role "ORGADMIN"
+* From "Node1" deactivate org "NWADMIN"'s node "Node3"
+* Check org "NWADMIN" has "Node3" with status "Deactivated"
```

Comment:
Minor: should you validate that org status has been updated on every node?

Reply:
Not required as the smart contract calls get executed on all nodes